server/utils/applications/getApplicationNoticeType.ts

```diff
@@ -0,0 +1,19 @@
+import { ApprovedPremisesApplication as Application } from '@approved-premises/api'
+import { differenceInDays, differenceInCalendarMonths } from 'date-fns'
+import { getArrivalDate } from './utils'
+import { DateFormats } from '../dateUtils'
+
+type ApplicationType = 'emergency' | 'short_notice' | 'standard'
+
+export const getApplicationNoticeType = (application: Application): ApplicationType => {
```

Comment:
What do you think of `noticeTypeFromApplication`? I know it goes against what I've done in the past but I think we should reserve `get` prefixes for a call to the API/session

Reply:
Good call 👍